src/library/scala/util/control/Exception.scala

```diff
@@ -13,24 +13,125 @@ package control
 import scala.reflect.{ ClassTag, classTag }
 import scala.language.implicitConversions
 
-
 /** Classes representing the components of exception handling.
- *  Each class is independently composable.  Some example usages:
+ *
+ *  Each class is independently composable.
+ *
+ *  This class differs from [[scala.util.Try]] in that it focuses on composing exception handlers rather than
+ *  composing behavior.  All behavior should be composed first and fed to a [[Catch]] object using one of the
+ *  `opt`, `either` or `withTry` methods. Taken together the classes provide a DSL for composing catch and finally
+ *  behaviours.
```

Comment:
Occurrences of each variation,

|behavior|176|
|behaviour|51|

Majority rule => behavior